src/System.Security.Principal/src/project.json

```diff
@@ -9,6 +9,11 @@
       "dependencies": {
         "Microsoft.TargetingPack.NETFramework.v4.6": "1.0.1"
       }
+    },
+    "uap10.1": {
+      "dependencies": {
+        "Microsoft.TargetingPack.Private.NETNative": "1.1.0-beta-24716-00"
```

Comment:
This should be building against System.Runtime like the CoreCLR version, not against the targeting pack.